pypeit/core/wavecal/templates.py

```diff
@@ -327,6 +334,55 @@ def main(flg):
         wfile = os.path.join(template_path, 'Keck_LRIS', 'R600_5000', 'MasterWaveCalib_B_1_01.json')
         build_template(wfile, slits, lcut, binspec, outroot, lowredux=False)
 
```

Comment:
MagE is a fixed format spectrograph which never changes. As such, we should be able to archive its wavelengths in  a PypeIt format. There should not be any need to import wavelength solutions from XIDL to solve for MagE. This may have very well been your starting point, but once you create a MagE wavelength solution for all of the orders, there is no need to archive this old solution or add this template stuff here. My suggestion is that we discuss what the best path forward is for archiving fixed format echelle wavelength solutions.  I disagree that any of this code here should be added to PypeIt. Perhaps to the development suite, but once you have a .json master file for MagE you should not need any of this code here.